README.md

```diff
@@ -1,223 +1,107 @@
-3. We have bundled a shell script that can generate test events. Get the source “writeKey” from our app dashboard and then run the following command. Run `cd scripts; ./generate-event <writeKeyHere> http://localhost:8080/v1/batch`. NOTE: `writeKey` is different from the `your_workspace_token` in step 2. Former is associated with the source while the latter is for your account.
-4. You can then login to your Google Analytics account and verify that events are delivered. Go to `MainPage->RealTime->Events`. `RealTime` view is important as the other dashboard can sometimes take 24-48 hrs to refresh.
-5. You can use our [Javascript][rudder-sdk-js-git-repo], [Android][rudder-sdk-android-git-repo] or [iOS][rudder-sdk-ios-git-repo] SDKs for sending events from your app.
+- **Extreme Scale**: One of our largest installations currently sends **300 Million** events/day with peak of **40K** req/sec, via a multi-node RudderStack setup
 
----
+- **Privacy and Security**: You can collect and store your customer data without sending everything to a third-party vendor. With RudderStack, you get fine-grained control over what data to forward to which analytical tool
 
-# RudderStack Config Generator
+- **Seamless Integration**: RudderStack currently supports integration with over 45 popular destination platforms such as Google Analytics, Amplitude, Mixpanel, Amazon S3,and more. Moreover, it also supports integration with popular data warehouses such as Snowflake, Redshift, and Google BigQuery
 
-Rudderstack has two components _control plane_ and _data plane_.
-Data plane reliably delivers your event data. Control plane manages the configuration of your sources and destinations.
-This configuration can also be read from a file instead of from Control plane, if you don't want to use our hosted control plane.
+- **Client-side SDK Support**: RudderStack offers client-side SDKs for [JavaScript](https://docs.rudderstack.com/sdk-integration-guide/getting-started-with-javascript-sdk), [Android](https://docs.rudderstack.com/sdk-integration-guide/getting-started-with-android-sdk), [iOS](https://docs.rudderstack.com/sdk-integration-guide/getting-started-with-ios-sdk), and [Unity](https://docs.rudderstack.com/sdk-integration-guide/getting-started-with-unity-sdk)
 
-Config-generator provides the UI to manage the source and destination configurations without needing to signup, etc.
-All the source and destination configuration stays on your local storage. You can export/import config to a JSON file.
+- **User-specified Transformation**: RudderStack gives you the power to filter or transform your events before sending them to the desired destinations
 
-## Setup
+# Key Features
 
-1. `cd utils/config-gen`
-2. `npm install`
-3. `npm start`
+- **Segment API Compatible**: RudderStack is Segment API and library compatible. So you don't need to change your app if you are using Segment.
```

Comment:
RudderStack is Segment API compatible. So you don't need to change your app if you are using Segment, just integrate RudderStack SDKs into your app.